MeshGeoToolsLib/BoundaryElementsSearcher.cpp

```diff
@@ -74,6 +96,22 @@ BoundaryElementsSearcher::getBoundaryElementsAtPoint(GeoLib::Point const& point)
     return _boundary_elements_at_point.back()->getBoundaryElements();
 }
 
+std::vector<std::pair<std::size_t, unsigned>> const&
+BoundaryElementsSearcher::getBulkIDsAtPoint(GeoLib::Point const& point)
+{
+    // Look for already saved points and return if found.
+    for (auto const& boundary_elements : _boundary_elements_at_point)
+    {
+        if (boundary_elements->getPoint() == point)
+            return boundary_elements->getBulkIDs();
```

Comment:
The STL version:
```c++
auto it = std::find_if(begin(_boundary_elements_at_point), end(_boundary_elements_at_point,
   [&point](auto const& boundary_element)  {
      return boundary_elements->getPoint() == point;
   });
if (it != _boundary_elements_at_point.end())
    return it->getBulkIDs();
```

... is not necessarily shorter

Reply:
It is not shorter and not substantially more readable. I would like to keep my implementation. Is this okay for you?